packages/manager/apps/pci-cold-archive/src/pages/containers/new/New.page.tsx

```diff
@@ -55,9 +55,14 @@ export default function ContainerNewPage() {
 
   const goBack = () => navigate('..');
 
+  useEffect(() => {
+    clearNotifications();
```

Comment:
I can't figure out the necessity of this code, it means that you want to clear notifications every time project changes? 

If so, why just not clearing notifications at unmount only?

Reply:
We need to clear notification when we arrive on this page not before to avoid notifications in double

Reply:
I see the point, so can we achieve that with a simple `componentDidMount` lifecycle, without any dependency in the dependency array? 
```suggestion
    useEffect(() => {
     clearNotifications();
   }, []);
```